### PR TITLE
add helper func to supplier init

### PR DESCRIPTION
**Pull request by diarmuidmcg**

### CODE REVIEW

**Comment by diarmuidmcg** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/miamtech/mealz-documentation/43?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#43** <a href="https://app.graphite.dev/github/pr/miamtech/mealz-documentation/43?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#42** <a href="https://app.graphite.dev/github/pr/miamtech/mealz-documentation/42?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @diarmuidmcg and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/shared/overview/supplierInit.mdx [hunk 1]
```diff
@@ -26,6 +26,10 @@ export function NewBasketService(props){
     if(props.platform == "android") { return <AndroidBasketService.NewBasketService  /> } else return  <IosBasketService.NewBasketService  />
 }
 
+export function HelperFunctionTransform(props){
```
Comment (ThibaultMiam) [inline docs/shared/overview/supplierInit.mdx:29]: Mapping function ?
your product to SupplierProduct ?

**Review by ThibaultMiam — CHANGES_REQUESTED** [review]

File: docs/shared/overview/supplierInit.mdx [hunk 2]
```diff
@@ -118,6 +122,14 @@ If at some point, you feel like products magically disappear from Mealz recipes,
 from the app basket while they should be, this is probably related to this part.
 :::
 
+### Helper Function to map between products
+
+More than likely, your Product object will be quite different than our `SupplierProduct`.
```
Comment (ThibaultMiam) [inline docs/shared/overview/supplierInit.mdx:127]: More than likely, your Product object will be quite different than our, then we have create a bridge model call `SupplierProduct`

**Review by ThibaultMiam — CHANGES_REQUESTED** [review]

File: docs/shared/overview/supplierInit.mdx (hunk 2 above)
Comment (diarmuidmcg) [reply docs/shared/overview/supplierInit.mdx]: c'est fait

**Comment by diarmuidmcg** [comment]
### Merge activity

* **Jun 28, 11:52 AM EDT**: @diarmuidmcg started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/miamtech/mealz-documentation/43).

**Pull request merged**